frontend/packages/helm-plugin/src/components/list-page/HelmChartRepositoryRow.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TableData } from '@console/internal/components/factory';
+import { ResourceKebab, ResourceLink, Timestamp, Kebab } from '@console/internal/components/utils';
+import { referenceFor, referenceForModel } from '@console/internal/module/k8s';
+import { HelmChartRepositoryModel, ProjectHelmChartRepositoryModel } from '../../models';
+
+const helmChartRepositoryReference = referenceForModel(HelmChartRepositoryModel);
+const projectHelmChartRepositoryReference = referenceForModel(ProjectHelmChartRepositoryModel);
+
+const HelmChartRepositoryRow = ({ obj }) => {
```

Comment:
Define the type for the `HelmChartRepositoryRow` component.

Reply:
Updated the PR